### PR TITLE
Azure .NET client Server Trust Groups tests

### PR DESCRIPTION
**Pull request by bojovicmatija**
Adding tests for  Server Trust Groups  .NET client. The client has already been generated, but tests have not been added.

### CODE REVIEW

File: sdk/sqlmanagement/Microsoft.Azure.Management.SqlManagement/src/Microsoft.Azure.Management.Sql.csproj [hunk 1]
```diff
@@ -7,7 +7,7 @@
 		<PackageId>Microsoft.Azure.Management.Sql</PackageId>
 		<Description>Azure SQL Management SDK library</Description>
 		<AssemblyName>Microsoft.Azure.Management.Sql</AssemblyName>
-		<Version>1.48.0-preview</Version>
+		<Version>1.49.0-preview</Version>
```
Comment (sopanic) [inline sdk/sqlmanagement/Microsoft.Azure.Management.SqlManagement/src/Microsoft.Azure.Management.Sql.csproj:10]: Was this on purpose? Same goes for the file below.

File: sdk/sqlmanagement/Microsoft.Azure.Management.SqlManagement/tests/ServerTrustGroupsTests.cs [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+				ResourceGroup resourceGroup = context.CreateResourceGroup(TestEnvironmentUtilities.DefaultLocationId);
+				Assert.NotNull(resourceGroup);
+				createManagedInstances(context, resourceGroup, managedInstanceNames);
+
+				IList<ServerInfo> groupMembers = createGroupMembers(sqlClient, resourceGroup.Name, managedInstanceNames);
+				IList<string> trustScopes = new List<string>() {"GlobalTransactions"};
+				ServerTrustGroup parameters = new ServerTrustGroup(groupMembers, trustScopes);
+
+				ServerTrustGroup serverTrustGroup = sqlClient.ServerTrustGroups.CreateOrUpdate(resourceGroup.Name, TestEnvironmentUtilities.DefaultLocationId, stgName, parameters);
+				Assert.NotNull(serverTrustGroup);
```
Comment (sopanic) [inline sdk/sqlmanagement/Microsoft.Azure.Management.SqlManagement/tests/ServerTrustGroupsTests.cs:85]: Consider adding some explanatory message when using Assert for easier debugging.

**Review by sopanic — APPROVED** [review]

**Comment by YalinLi0312** [comment]
Hi @bojovicmatija 
* If the client has checked in, can you provide the PR for it? It should have a related swagger PR, and includes metadata updates.
* Can you resolve the comments in this PR?

File: sdk/sqlmanagement/Microsoft.Azure.Management.SqlManagement/src/Microsoft.Azure.Management.Sql.csproj (hunk 1 above)
Comment (bojovicmatija) [reply sdk/sqlmanagement/Microsoft.Azure.Management.SqlManagement/src/Microsoft.Azure.Management.Sql.csproj:10]: Yes, I believe version should be updated with changes

**Comment by bojovicmatija** [comment]
Hi @YalinLi0312, 
Client was checked in and the [PR ](https://github.com/Azure/azure-sdk-for-net/pull/15981) for it is #15981. Comments are resolved.

File: sdk/sqlmanagement/Microsoft.Azure.Management.SqlManagement/src/Microsoft.Azure.Management.Sql.csproj (hunk 1 above)
Comment (YalinLi0312) [reply sdk/sqlmanagement/Microsoft.Azure.Management.SqlManagement/src/Microsoft.Azure.Management.Sql.csproj:10]: Same doubt here, is there any reason to update versions with just tests updates?

Comment (bojovicmatija) [reply sdk/sqlmanagement/Microsoft.Azure.Management.SqlManagement/src/Microsoft.Azure.Management.Sql.csproj:10]: You're right. I will revert this. I thought that it should be updated with every change.

**Review by YalinLi0312 — APPROVED** [review]

**Pull request merged**